deployments/aws/templates/stack-services.yaml

```diff
@@ -210,48 +210,51 @@ Resources:
     Type: AWS::ElasticLoadBalancingV2::TargetGroup
     DependsOn: ServicesLoadBalancer
     Properties:
-      HealthCheckIntervalSeconds: 30
       HealthCheckProtocol: HTTP
+      HealthCheckPath: /.well-known/veraison/provisioning
+      HealthCheckIntervalSeconds: 30
       HealthCheckTimeoutSeconds: 15
       HealthyThresholdCount: 5
+      UnhealthyThresholdCount: 3
       Matcher:
-        HttpCode: '401'
+        HttpCode: '200'
```

Comment:
Is this sufficient?  I thought we also needed `"service-state": “READY”`.

Reply:
This appears to be sufficient.  This fails if it can't fetch the media types, so I think, in parctice  `200` here implies services state READY.